app/views/admin/users/_member_index.html.erb

```diff
@@ -42,7 +42,7 @@
                   </div>
                   <div>
                     <dt class="color-base-60 fw-normal">Joined on</dt>
-                    <dd><%= user.registered_at.strftime("%d %b, %Y") %></dd>
+                    <dd><%= user.registered_at&.strftime("%d %b, %Y") %></dd>
```

Comment:
This is just a precaution even though we're only showing registered users on this page.